lib/changeNIDMtoSPM.m

```diff
@@ -78,6 +96,19 @@
     [~, name, ext] = fileparts(designMatrixFilename);
     xXtemp.xKXs.X = csvread(fullfile(filepathTemp, [name, ext]));
     
+    %Record the SVD data.
```

Comment:
Same comment as above. Can you (very) shortly explain why those changes were made?

Reply:
If I am completely honest, I am not sure why I made these changes on this branch. I think I must have thought I was on a different branch. They are useful small changes though in that they result in the SPM object generated by changeNIDMtoSPM more closely resembling that generated by normal SPM output. I'd still recommend keeping them!

Reply:
Okay! That's fine.